engine/src/test/java/org/camunda/bpm/engine/test/api/task/TaskServiceTest.java

```diff
@@ -2228,14 +2226,48 @@ public void testDeleteTaskAttachmentWithNullParameters() {
   }
 
   @Test
-  public void testDeleteTaskAttachmentWithTaskIdNull() {
-    int historyLevel = processEngineConfiguration.getHistoryLevel().getId();
-    if (historyLevel> ProcessEngineConfigurationImpl.HISTORYLEVEL_NONE) {
-      try {
-        taskService.deleteTaskAttachment(null, "myAttachmentId");
-        fail("expected process engine exception");
-      } catch(ProcessEngineException e) {}
-    }
+  @RequiredHistoryLevel(ProcessEngineConfiguration.HISTORY_NONE)
+  public void testDeleteTaskAttachmentThatDoesNotExist() {
+    assertThatThrownBy(() -> taskService.deleteTaskAttachment(null, "attachmentDoesNotExist")).isInstanceOf(
```

Comment:
Yes we already have `testDeleteTaskAttachmentWithNullParameters` where both taskId and attachmentId are null, however deleting an attachment with null attachmentId is essentially the same exception as `testDeleteTaskAttachmentThatDoesNotExist` because no attachment exists with that id.